cmd/miqt-uic/ui2go.go

```diff
@@ -595,28 +688,32 @@ func generateWidget(w UiWidget, parentName string, parentClass string, useQt6 bo
 		}
 	}
 
-	if w.Class == "QDialogButtonBox" {
-		// TODO make this using a native connection instead of a C++ -> Go -> C++ roundtrip
-		ret.WriteString(`ui.` + w.Name + `.OnAccepted(` + RootWindowName + ".Accept)\n")
-		ret.WriteString(`ui.` + w.Name + `.OnRejected(` + RootWindowName + ".Reject)\n")
+	return ret.String(), nil
+}
+
+func normalizeSignalName(s string) string {
+	if s == "" {
+		return ""
 	}
 
-	return ret.String(), nil
+	s = strings.TrimSuffix(s, `()`)
+	return strings.ToUpper(string(s[0])) + s[1:]
```

Comment:
The signals aren't always empty, e.g.:
```
<signal>valueChanged(int)</signal>
```

Here's what I put into a similar method:
```suggestion
	if idx := strings.IndexByte(s, '('); idx != -1 {
		return s[:idx]
	}
	return s
```

Reply:
Thanks for the snippet! I'll do it that way.

This will work as long as the signal and slot have the same number of arguments. It is possible for the signal to have more arguments than the slot, so it might need changes later.